cfme/configure/access_control/__init__.py

```diff
@@ -396,6 +399,33 @@ def create(self, name=None, credential=None, email=None, group=None, cost_center
         view.browser.refresh()
         return user
 
+    def delete(self, *users):
+        """
+        Delete user(s) using the Access Control EVM Users checklist
+        """
+        users = list(users)
+        delete_toolbar_txt = 'Delete selected Users'
+        full_name_header = 'Full Name'
+        delete_success_msg = 'EVM User "{}": Delete successful'
+        delete_blocked_msg = 'Default EVM User "{}" cannot be deleted'
+
+        view = navigate_to(self, 'All')
+        for user in users:
+            try:
+                find_row_kwargs = {full_name_header: user.name}
+                # This will refresh the page to find the item so you won't have multiple items
+                # selected per page
+                row = view.paginator.find_row_on_pages(view.table, **find_row_kwargs)
+                row[0].check()
+                view.toolbar.configuration.item_select(delete_toolbar_txt, handle_alert=True)
+                # Check that deleting the user wasn't blocked
+                view.flash.assert_message(delete_blocked_msg.format(user.name))
+                raise RBACOperationBlocked(view.flash.messages[0].text)
```

Comment:
**REQUIRED:** If you could add a comment here to explain the logic that would help make this more understandable.   Per our discussion offline I thought you were raising this exception because there was some problem but the code looked like there wasn't a problem, and I was certainly wrong.   So comment would make this clearer.